treeherder/model/derived/jobs.py

```diff
@@ -1937,15 +1937,24 @@ def _load_log_urls(self, log_placeholders, job_id_lookup,
                 log_placeholders[index][0] = job_id
                 log_placeholders[index].append(time_now)
                 task = dict()
-                task['job_guid'] = job_guid
-                task['log_url'] = log_ref[2]
-                task['result_set_id'] = result_set_id
-                task['check_errors'] = True
-                if result != 'success':
-                    task['routing_key'] = 'parse_log.failures'
+
+                if log_ref[1] == 'structured-raw':
+                    # don't parse structured logs for passing tests
```

Comment:
I wonder if some of our tooling will want to use the output from passing tests too? (eg "compare two test runs" to compare a passing and failing run of a certain job type). Though presume there would be quite a perf hit.